wpt-status/index.html

```diff
@@ -0,0 +1,387 @@
+---
+title: "WPE Specification Support"
+layout: page
+tags: [devs]
+data: { dateless: "true" }
+htmlSitemapOrder: -2
+sitemapChangeFrequency: weekly
+---
+
+<style>
+body { font-family: sans-serif; }
+h1 { margin-block-end: 0; }
+
+#header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#stickyheader {
+  position: sticky;
+  top: 4.5em;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  gap: 0 2em;
+  padding-block: 1em;
+  z-index: 1;
+  background: #FFF;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+#stickyheader > label {
+  line-height: 1.6;
+}
+
+#stickyheader + p {
+     margin-block-start: 0em;
+}
+
+#version-select {
+  margin-inline: 0.67em 0;
+}
+
+#results-table {
+   border-collapse: collapse;
+   width: 100%;
+   margin-block: 1.25em 1em;
+   table-layout: fixed;
+}
+#results-table :is(th, td) {
+   border: 1px solid #ccc;
+   padding-inline: 0.5em;
+   padding-block: 0.5em 0.4em;
+   text-align: left;
+}
+#results-table thead th {
+   background-color: #f4f4f4;
+   background-repeat: no-repeat;
+   background-position: 0 0, 100% 100%;
+   background-size: 100% 1px;
+   background-image: linear-gradient(90deg,#0003,#0003), linear-gradient(90deg,#0005,#0005);
+}
+#results-table thead {
+   position: sticky;
+   top: 7.5em;
+}
+#results-table tbody a[href] {
+   text-decoration-color: color-mix(in hsl, currentColor 60%, transparent);
+   text-underline-offset: 0.15em;
+}
+#results-table tbody td.barred {
+   background-image: linear-gradient(90deg, hsl(calc(var(--dataHue) * 1.2) 70% 90%) 0, hsl(calc(var(--dataHue) * 1.2) 70% 70%) 100%);
+   background-size: var(--dataPct) 100%;
+   background-repeat: no-repeat;
+   text-shadow:
+   0 0 0.25em hsl(0, 0%, 93.7%),
+   0 0 0.25em hsl(0, 0%, 93.7%),
+   0 0 0.25em hsl(0, 0%, 93.7%),
+   0 0 0.25em hsl(0, 0%, 93.7%),
+   0 0 0.25em hsl(0, 0%, 93.7%),
+   0 0 0.15em hsl(0, 0%, 93.7%),
+   0 0 0.15em hsl(0, 0%, 93.7%),
+   0 0 0.15em hsl(0, 0%, 93.7%),
+   0 0 0.15em hsl(0, 0%, 93.7%),
+   0 0 0.15em hsl(0, 0%, 93.7%);
+}
+#results-table tbody td small {
+   margin-inline: 0.33ch 0;
+}
+
+#support-combo-filter {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 0.5em 1em;
+  background: inherit;
+}
+#support-combo-filter legend {
+  grid-column: span 2;
+}
+
+details {
+  margin-block-end: 1px;
```

Comment:
```suggestion
  padding-block-end: 1px;
```

Reply:
I mistyped in Matrix; it was supposed to be a block-end padding instead of a margin.  You copied what I typed over accurately, to be clear.  I just had a brain cramp when I posted there. 🤦

Reply:
change added, thanks